source/commands.txt

```diff
@@ -0,0 +1,114 @@
+.. _vsce-commands:
+
+============================
+MongoDB for VS Code Commands
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This document lists the VS Code commands available for
+|vsce|. To open the Command Palette and execute these commands:
+
+.. include:: /includes/list-tables/command-palette-options.rst
+
+Connection Commands
+-------------------
+
+Use these commands to manage connections to your MongoDB deployments.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Command
+     - Description
+
+   * - ``Connect``
+     - Connect to your MongoDB deployment by filling in your connection
+       information in the connection dialog. To learn more about
+       connecting to a deployment, see :ref:`vsce-connect`.
+
+   * - ``Connect with Connection String``
+     - Connect to your MongoDB deployment by pasting your connection
+       string directly into the command palette. To learn more about
+       connecting to a deployment, see :ref:`vsce-connect`.
+
+   * - ``Disconnect``
+     - If you are currently connected to a MongoDB deployment,
```

Comment:
More of an implementation question, but we state 'deployments' up high, but 'deployment' here - do we support multiple connections? If so, how do I use this command to specify *one* deployment? Or is the meaning here that we disconnect from the *selected* deployment (i.e. i have to select a deployment, *then* run Disconnect)

Reply:
I think we need to clarify between creating connections and having an 'Active' connection. I'm assuming 'Active' connection is the one selected in the MongoDB View inside VSCode?

i.e. creating a connection adds a new entry to the MongoDB View. 
Selecting a connection activates it.
The Disconnect/Remove command removes the active/selected connection.

Reply:
All good points. Yes, 'Active' is referring to the one which is currently selected. I added some more detail to the introduction of this section.